datafeeder/datafeeder.properties

```diff
@@ -95,6 +95,10 @@ datafeeder.publishing.backend.geoserver.dbtype=postgis
 datafeeder.publishing.backend.geoserver.preparedStatements=true
 #<schema> is a placeholder to be replaced by the actual schema computed from the "sec-org" request header
 datafeeder.publishing.backend.geoserver.schema=<schema>
+#<workspacename> is a placeholder to be replaced by the lowercase "sec-org" request header
+datafeeder.publishing.backend.geoserver.workspacename=<workspacename>
+#<storename> is a placeholder to be replaced by the lowercase "sec-org" request header prefixed by 'datafeeder_'
+datafeeder.publishing.backend.geoserver.storename=<storename>
```

Comment:
I might be a bit late, this is kind of an afterthought - sorry about that, but would it be clearer with:
```
datafeeder.publishing.backend.geoserver.schema=<org_id>
datafeeder.publishing.backend.geoserver.workspacename=<org_id>
datafeeder.publishing.backend.geoserver.storename=datafeeder_<org_id>
```